src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -134,8 +137,11 @@ static void HandleNodeIdResolve(void * context, DnssdService * result, const Spa
     }
     nodeData.resolutionData.numIPs = addressesFound;
 
+    ChipLogDetail(Discovery, "Node with operational ID %s resolved. TXT:", result->mName);
```

Comment:
Isn't it displaying almost the same info as `LogNodeIdResolved` below?

Reply:
Hmm.  I guess it is.  The latter does not log T (which I can add) and does not log unknown TXT keys, but that's probably OK.